tests/Fixtures/ChoiceFactoryMock.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Fixtures;
+
+use WMDE\Fundraising\Frontend\BucketTesting\FeatureToggle;
+use WMDE\Fundraising\Frontend\Factories\UnknownChoiceDefinition;
+
+/**
+ * @license GNU GPL v2+
+ */
+class ChoiceFactoryMock {
+
+	private $featureToggle;
+
+	public function __construct( FeatureToggle $featureToggle ) {
+		$this->featureToggle = $featureToggle;
+	}
+
+	public function someMockBucketSelection(): void {
```

Comment:
In the spirit of how the real `ChoiceFactory` works, could you return strings instead of `echo`ing stuff? Or is there a specific intention behind not returning anything to stress the fact that this is a test double?

Reply:
I totally did not keep the spirit of the real ChoiceFactory in mind since I figured it does not really matter for code that is never actually executed and only used for static analysis, but I'll quickly change that anyway!